lib/cluster.js

```diff
@@ -261,7 +261,7 @@ function masterInit() {
     assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
            'Bad cluster.schedulingPolicy: ' + schedulingPolicy);
 
-    var hasDebugArg = process.execArgv.some(function(argv) {
+    var hasDebugArg = process.execArgv.some(function hasDebugTest(argv) {
```

Comment:
https://github.com/nodejs/node/issues/8913#issuecomment-257042842
second code section in this comment.
i see the first code section was edited by trott.